set.mm

```diff
@@ -279710,7 +279713,7 @@ we show (in ~ tgcl ) that ` ( topGen `` B ) ` is indeed a topology (on
   ${
     $d g j v x J $.  $d g v N $.  $d g P $.  $d g v x S $.  $d g j v x X $.
     neifval.1 $e |- X = U. J $.
-    $( The neighborhood function on the subsets of a topology's base set.
+    $( Value of the neighborhood function on the subsets of the base set.
        (Contributed by NM, 11-Feb-2007.)  (Revised by Mario Carneiro,
```

Comment:
Why is "topology" removed from the comment? What is "the base set" then? I would like to keep "of a topology's base set".

Reply:
It's obvious, but ok, I will leave it. By the way, I don't know a single math book using that terminology, or defining a topology before "its base set".

I'll write "the base set of a topology" since apostrophes are for persons (that's what was bugging me in the comments and the main reason I removed it in the first place).

Reply:
I tried to research "apostrophes are for persons" but the best reference materials (e.g. https://www.cambridge.org/core/books/cambridge-grammar-of-the-english-language/A78402ABF5176AD283494180BCA2046F ) are behind a paywall and the various pages I did find said something like "a thing" (or something more general than persons) and proceed to give examples that are exclusively persons or animals, so that's a bit inconclusive. Anyway, I don't object to "the base set of a topology" so we can go with that.

Reply:
Genitives: you're right, I should have added animals. I learned in school that you can use it when it answers the question "whose is it" and we can answer "it's his/hers/theirs". I found the page: https://www.colanguage.com/genitive-english for what it's worth. But as you said, usage changes and apostrophes are used in other cases as well.  English does not have a "spec" like ISO C !
Side remark: I may sound pedantic, but actually I am more comfortable with strict grammar rules precisely because I am less comfortable with the English language and its evolving usage. Stricter rules make it easier (same as in math, where a beginning mathematician would be puzzled by abuses of language that mathematicians make every day).